TelepresenceBot/core/src/main/java/com/novoda/tpbot/human/HumanPresenter.java

```diff
@@ -42,6 +47,7 @@ public void update(Result result) {
             if (result.isError()) {
                 humanView.onError(result.exception().get().getMessage());
             } else {
+                lastServerPersistence.saveLastConnectedServer(serverAddress);
```

Comment:
🤔  interesting, I like the idea. Do you have an example? I would like to try the idea, I can't think of any drawbacks without trying it 👅